src/cmds/net/arping.c

```diff
@@ -114,15 +115,39 @@ int main(int argc, char **argv) {
 	from.s_addr = in_dev->ifa_address;
 	strncpy(from_b, inet_ntoa(from), sizeof(from_b) - 1);
 	printf("ARPING %s from %s %s\n", dst_b, from_b, in_dev->dev->name);
+
 	for (i = 1; i <= cnt; i++) {
 		neighbour_del(ETH_P_IP, &dst, in_dev->dev);
+
+		if (gettimeofday(&t1, NULL) == -1) {
```

Comment:
> Do you mean adding in 1 sec delay between send_request() and neighbour_get_haddr(), or 1 sec delay after neighbour_get_haddr()?

I mean to add delay right before `gettimeofday` at line 121. Because if you will add delay between send_request() and neighbour_get_haddr() the actual time will be incorrect.